cloudferrylib/os/actions/filter_similar_vms_from_dst.py

```diff
@@ -65,7 +65,9 @@ def find_similar_instances(self):
         self.similar_isntances = collections.defaultdict(set)
         self.conflict_instances = collections.defaultdict(set)
         for tenant_id, ip_to_id in titii_src.items():
-            tenant_new_id = self.tenant_id_to_new_id[tenant_id]
+            tenant_new_id = self.tenant_id_to_new_id.get(tenant_id, None)
+            if tenant_new_id is None:
+                continue
```

Comment:
This change introduces inconsistency: for most objects `[<cloud>] tenant` config value is used if object's tenant was deleted, while this patch simply ignores objects which belong to deleted tenant. I think correct resolution should be based either on properly maintaining contents of `src_cloud['identity_info']['tenants']['meta']['new_id']` values or reusing `cloudferrylib.os.identity.keystone.get_dst_user_from_src_user_id()` or something similar.